packages/components/datetime/DateTime.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: 'DateTime'
+type: 'component'
+status: 'stable'
+slug: /components/datetime/
+github: 'https://github.com/contentful/forma-36/tree/master/packages/components/components/datetime'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/components-datetime--basic'
+---
```

Comment:
please add `typescript:` metadata and use PropsTable to visualize props

Reply:
got it!